components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaDeviceConstellationTest.kt

```diff
@@ -232,6 +239,68 @@ class FxaDeviceConstellationTest {
         assertEquals(ConstellationState(currentDeviceExpired.into(), listOf(testDevice2.into())), constellation.state())
     }
 
+    @Test
+    @ExperimentalCoroutinesApi
+    fun `polling for commands with real account`() = runBlockingTest {
+        val testScope = GlobalScope
+
+        testScope.launch {
+            // Initialize things so the megazord is happy
+            WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+            Megazord.init()
+
+            // Create a test account
+            val testAccount = spy(NativeFirefoxAccount(
```

Comment:
I think it will be more valuable to attempt to switch existing tests in this class to use `NativeFirefoxAccount` (instead of the mock that they're currently using).